src/input_common/motion_input.h

```diff
@@ -29,8 +29,8 @@ class MotionInput {
     void EnableReset(bool reset);
     void ResetRotations();
 
-    void UpdateRotation(u64 elapsed_time);
-    void UpdateOrientation(u64 elapsed_time);
+    void UpdateRotation(const u64 elapsed_time);
```

Comment:
```suggestion
    void UpdateRotation(u64 elapsed_time);
```

Top-level const on a value type in the header doesn't do anything. It can be left on the parameter in the cpp file though.